hdf/src/mfanf.c

```diff
@@ -226,7 +226,7 @@ nafannlist(intf *an_id, intf *atype, intf *etag, intf *eref, intf alist[])
 
     /* create annlist with true int32s to maintain compatibility
     ** with machines that allocate less than 32 bits per int. */
```

Comment:
Are there any machines left? I suppose maybe some IoT nodes that can't even hold a HDF file in the first place.

Reply:
> What is the reason for removing HD prefix? All these years, we had to make sure we used it... And, HDF5 is still using it.

It's to paper over platform differences and as a way to allow injection of alternative functionality. In practice, they have usually been defined to the standard C call. We're removing it from HDF5 as well so our code looks like normal C and we can stop hassling people about using the prefix. Most other software does not do something like this.